src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -200,10 +206,12 @@ public void teardown() throws Exception {
 
     dataRepoFixtures.resetConfig(steward);
 
-    if (snapshotIds != null) {
-      for (UUID snapshotId : snapshotIds) {
-        dataRepoFixtures.deleteSnapshot(steward, snapshotId);
-      }
+    for (UUID snapshotAccessRequestId : snapshotAccessRequestIds) {
+      samFixtures.deleteSnapshotAccessRequest(steward, snapshotAccessRequestId);
```

Comment:
Is this a temporary state util we have a `SnapshotBuilderService` API that deletes requests? Because I think that would need to do this too.

Reply:
Right instead of creating an API in this ticket, I just call Sam directly to delete the resource there, then after that the snapshot can be successfully deleted as well